user-guides/channel-actions/README.md

```diff
@@ -1,69 +1,73 @@
 # Channel Actions
 
-With channels you can also do more than merely post messages and upload files, you can interact in various ways through the channel actions menu. Channel actions are located to the right of the channel title. Some of the menus are accessible by clicking on the three dots at the end of the channel header.
+Inside channels, you can also do more than post messages and upload files. Increase your interaction with messages through Channel actions.
+
+## Find channel actions
+
+To find channel actions, go to the right of the channel title and click the menu.
 
 _Channel actions vary depending on the server configuration, so the items listed below might not all be visible in your installation._
 
-## Favorites
+## Favorite
 
 To help organize and prioritize your communications, you can **favorite** (or **star**) channels, private groups, and DMs.
 
-To favorite a channel, click on the star icon to the left of the channel header (next to the room name).
+To favorite a channel, click the **star** icon to the left of the channel header.
 
-Favoriting a channel moves it to a "Favorites" section on the channels list, making it easier to access and find.
+Favoriting a channel moves it to a **Favorites** section on the channels list, making it easier to access and find.
 
 ## Channel Info
 
-Here you can see all kinds of information about the current channel. If you have the proper permissions, this information can be edited. These are the channel info options:
+Here you can see all kinds of information about the current channel. If you admin permissions, you can edit this information. These are the channel info options:
```

Comment:
`If you admin permissions` shouldn't be `if you have admin permissions`?

Reply:
Fixed!